### DISCUSSION TITLE
[Question] Is there a way of using upload_to_play_store without API ACCESS?

### DISCUSSION THREAD

**Discussion by nriccar** [fastlane/fastlane#20144] [Q&A]
Hello,

First of all, thank you for your time.

Im currently implementing Fastlane on a project in which I don't have access to the Account Holder. I implemented fastlane in the past but always with Google API ACCESS, and I was wondering if there's any way of using upload_to_play_store with username and password, similar to Apple Store Connect app-specific password authentication.

Thanks for reading!

**Comment by 1972chris** [comment]
Hello are you referring too my repo on GitHub. If so I will help or try to help. Thankyou
